Dockerfile

```diff
@@ -1,9 +1,16 @@
 FROM node:16 AS build
 WORKDIR /app
+
 COPY . .
+
 RUN yarn install --production && \
-  yarn run build
+  yarn build
+
+
+FROM gcr.io/distroless/nodejs:16
+
+COPY --from=build /app/build /app/assets /app/node_modules /
+
+WORKDIR /app/build
 
-FROM gcr.io/distroless/nodejs:10
-COPY --from=build /app/build /app/assets /app/node_modules /app/
-CMD [ "/app/build/index.js" ]
+ENTRYPOINT [ "index.js" ]
```

Comment:
`CMD` is most appropriated here since we could need to overwrite the existing command for debug purposes, which is not possible by using `ENTRYPOINT`